modules/xmpp/Caps.js

```diff
@@ -260,7 +260,6 @@ export default class Caps extends Listenable {
     _notifyVersionChanged() {
         // update the version for all rooms
         this.rooms.forEach(room => this._fixChatRoomPresenceMap(room));
-        this.submit();
```

Comment:
What did this submit do and why can it be removed?

Reply:
So the flow is addFeature/removeFeature -> _generateVersion -> _notifyVersionChanged, and we already control calling submit in addFeature/removeFeature. So we basically can call submit 2 times. And this is the only usage of _generateVersion and _notifyVersionChanged.